awscli/botocore/configprovider.py

```diff
@@ -89,6 +90,12 @@
         'use_fips_endpoint',
         'AWS_USE_FIPS_ENDPOINT',
         None, utils.ensure_boolean),
+    'ignore_configured_endpoint_urls': (
+        'ignore_configured_endpoint_urls',
+        'AWS_IGNORE_CONFIGURED_ENDPOINT_URLS',
+        None,
+        utils.ensure_boolean,
+    ),
```

Comment:
We will still need to implement the `__copy__()` method on the `ConfigValueStore` store